daprdocs/content/en/developing-applications/integrations/Azure/azure-authentication/authenticating-azure.md

```diff
@@ -9,38 +9,41 @@ aliases:
 weight: 10000
 ---
 
-Most Azure components for Dapr support authenticating with Microsoft Entra ID. Thanks to this:
```

Comment:
@msfussell this is a bit of a rework but Alex and I chatted and think this flows better with the document